lib/blogs_api/user/delete.ex

```diff
@@ -0,0 +1,23 @@
+defmodule BlogsApi.User.Delete do
+  @moduledoc """
+  Função que encontra e deleta o id no banco de dados
+  """
+  alias BlogsApi.{User, Repo}
+  alias Ecto.UUID
+
+  def call(id) do
+    case UUID.cast(id) do
+      :error -> {:error, "Invalid ID format"}
+      {:ok, uuid} -> delete(uuid)
+    end
+  end
+
+  defp delete(uuid) do
+    case fetch_user(uuid) do
+      nil -> {:error, "Usuário não encontrado!"}
```

Comment:
Ao propagar esse erro com uma `string`, no seu `fallback controller` ele será retornado como um `400` para `not found`. Esse é o comportamento esperado segundo a especificação?

Reply:
Oi Chagas, é isso mesmo o que o requisito espera!!! :D